docs/src/cli/sample_inverted_index.md

```diff
@@ -5,3 +5,23 @@
 ```
 <!-- cmdrun ../../../build/bin/sample_inverted_index --help -->
 ```
+
+## Description
+Creates a smaller inverted index from an existing one by sampling postings or documents. The purpose of this tool is to reduce time and space requirements while preserving the main statistical properties of the original collection, making it useful for faster experiments and debugging.
+
+### Sampling strategy (`-t, --type`)
+
+- `random_postings`: keep random occurrences per posting list (not whole posting lists).
+- `random_docids`: keep all postings belonging to a random subset of documents.
+
+## Examples
+
+### Keep ~25% of postings
+```bash
+sample_inverted_index -c path/to/inverted -o path/to/inverted.sampled -r 0.25 -t random_postings
```

Comment:
In a way, it is even more important to keep the width small in code because it won't be formatted, so you'll need to scroll horizontally and that's never a good UX.

So I would break it down like so:

```bash
first line that is under 80 characters \
    continue after backslash
```

By adding backslash, you make it possible to copy and paste the command.

But seeing this is documentation, we may want to highlight each parameter or at least the important ones, perhaps even with a comment if necessary (within reason).

```bash
sample_inverted_index \
    -c path/to/inverted \
    -o path/to/inverted.sampled \
    -r 0.25 \
    -t random_postings
```

Now we can see each argument clearly without having to parse a long line of text.